kuma/core/tests/test_views.py

```diff
@@ -152,6 +152,15 @@ def test_not_possible_to_set_non_locale_cookie(self):
         # No language cookie should be saved as `foo` is not a supported locale
         assert not language_cookie
 
+    def test_setting_language_cookie_age(self):
```

Comment:
@MatonAnthony I think you'll first have to rebase your branch on master, as the `LanguageCookieTest` class no longer exists. Once you rebase on the latest master, you could fold this check:
```
assert language_cookie['max-age'] == settings.LANGUAGE_COOKIE_AGE
```
into the existing test `test_setting_language_cookie_working`.